app/services/waste_exemptions_engine/generate_renew_token_service.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require_relative "concerns/can_process_jwt_tokens"
+
+module WasteExemptionsEngine
+  class GenerateRenewTokenService < BaseService
+    include CanProcessJwtTokens
+
+    def run(registration:)
+      payload = {
+        data: {
+          registration_id: registration.id
+        },
+        exp: exp
+      }
+
+      registration.renew_token = generate_token(payload)
+
+      registration.save!
+    end
+
+    private
+
+    def exp
+      # Expires in 60 days
+      Time.now.to_i + 60 * 24 * 3600
```

Comment:
We need to get the 60 days from an env var. Though we're unlikely to change it in the life of the app, being able to configure it differently will help us with testing (QA rather than unit testing).